### ISSUE TITLE
vmware-iso xenial.json build fail

### ISSUE THREAD

**Issue by InfamousSYN** [InfamousSYN/packer#1]
**Error:**
```
    vmware-iso: + echo ==> Disk usage after cleanup
    vmware-iso: + df -h
    vmware-iso: Filesystem                              Size  Used Avail Use% Mounted on
    vmware-iso: udev                                    981M     0  981M   0% /dev
    vmware-iso: + sync
    vmware-iso: tmpfs                                   201M  5.9M  195M   3% /run
    vmware-iso: /dev/mapper/vagrant--xenial64--vg-root   36G  4.1G   31G  12% /
    vmware-iso: tmpfs                                  1001M     0 1001M   0% /dev/shm
    vmware-iso: tmpfs                                   5.0M     0  5.0M   0% /run/lock
    vmware-iso: tmpfs                                  1001M     0 1001M   0% /sys/fs/cgroup
    vmware-iso: /dev/sda1                               472M  109M  339M  25% /boot
    vmware-iso: tmpfs                                   201M     0  201M   0% /run/user/1000
==> vmware-iso: Gracefully halting virtual machine...
    vmware-iso: Waiting for VMware to clean up after itself...
==> vmware-iso: Deleting unnecessary VMware files...
    vmware-iso: Deleting: output-vmware-iso\564d5619-ef9c-993d-496d-62ba4ae42b00.vmem
==> vmware-iso: Deleting output directory...
Build 'vmware-iso' errored: remove output-vmware-iso\564d5619-ef9c-993d-496d-62ba4ae42b00.vmem: Access is denied.

==> Some builds didn't complete successfully and had errors:
--> vmware-iso: remove output-vmware-iso\564d5619-ef9c-993d-496d-62ba4ae42b00.vmem: Access is denied.

==> Builds finished but no artifacts were created.
```

**Comment by InfamousSYN** [comment]
The issue is caused by converting the Xenial build from a server base to a desktop base.

**Comment by InfamousSYN** [comment]
Fixed in commit [dc66198](https://github.com/InfamousSYN/packer/commit/dc66198c6bff621233760ecee834dec53c4c689b)
